public/app/features/integrated-alerting/components/Table/Table.tsx

```diff
@@ -43,16 +43,7 @@ export const Table: FC<TableProps> = ({ pendingRequest, data, columns, emptyMess
               ))}
             </thead>
             <tbody {...getTableBodyProps()} data-qa="table-tbody">
-              {rows.map(row => {
```

Comment:
I think we should leave this by default if no children is provided so we don't need to add it when using the component.

Reply:
Yep, good one !